rhel/SPECS/pmm-server.spec

```diff
@@ -47,6 +47,7 @@ mv nginx.conf %{buildroot}%{_sysconfdir}/nginx/conf.d/pmm.conf
 mv nginx-ssl.conf %{buildroot}%{_sysconfdir}/nginx/conf.d/pmm-ssl.conf
 install -d %{buildroot}%{_datadir}/percona-dashboards
 mv import-dashboards.py %{buildroot}%{_datadir}/percona-dashboards/import-dashboards.py
+mv import-dbass-dashboard.py %{buildroot}%{_datadir}/percona-dashboards/import-dbass-dashboard.py
```

Comment:
I don't see `import-dbass-dashboard.py` in the other PR

Reply:
I have a discussion with @askomorokhov and we came to conclusion that it's better to omit using an additional files.
So this PR is outdated.